.run/Build agent.run.xml

```diff
@@ -0,0 +1,21 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Build agent" type="ShConfigurationType" folderName="agent">
+    <option name="SCRIPT_TEXT" value="" />
+    <option name="INDEPENDENT_SCRIPT_PATH" value="true" />
+    <option name="SCRIPT_PATH" value="$PROJECT_DIR$/.run/bash_runner.sh" />
+    <option name="SCRIPT_OPTIONS" value="" />
+    <option name="INDEPENDENT_SCRIPT_WORKING_DIRECTORY" value="true" />
+    <option name="SCRIPT_WORKING_DIRECTORY" value="$PROJECT_DIR$" />
+    <option name="INDEPENDENT_INTERPRETER_PATH" value="true" />
+    <option name="INTERPRETER_PATH" value="/bin/zsh" />
+    <option name="INTERPRETER_OPTIONS" value="" />
+    <option name="EXECUTE_IN_TERMINAL" value="true" />
+    <option name="EXECUTE_SCRIPT_FILE" value="true" />
+    <envs>
+      <env name="SCRIPT_TO_RUN" value=".run/build.sh" />
+      <env name="BUILD_COMMAND" value="invoke agent.build" />
+      <env name="REMOTE_MACHINE_IP" value="" />
```

Comment:
consider to make the remote_machine_ip as an external environment variable
otherwise everyone to change it will have to struggle and remove the diff from the commits